packages/store/src/internal/state-factory.ts

```diff
@@ -56,10 +54,14 @@ export class StateFactory {
     private _internalStateOperations: InternalStateOperations
   ) {}
 
+  private _states: MappedStore[] = [];
```

Comment:
We should stick to the established convention that private fields are declared at the top.

Reply:
Fixed in my commit

Reply:
IDEA automatically formats, then configure tslint

Reply:
Yeah, your auto formatting keeps doing things it shouldn't 😕